eclipse-collections/src/main/java/org/eclipse/collections/impl/factory/Arrays.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2017 BNY Mellon.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.impl.factory;
+
+import org.eclipse.collections.impl.list.fixed.ArrayAdapter;
+
+/**
+ * This class can be used to create instances of ArrayAdapter.
+ * <p/>
+ * <pre>
+ * ArrayAdapter&lt;String&gt; arrayAdapter = Arrays.asList("a", "b", "c");
+ * </pre>
+ */
+public final class Arrays
+{
+    private Arrays()
+    {
+        throw new AssertionError("Suppress default constructor for noninstantiability");
+    }
+
+    @SafeVarargs
+    public static <T> ArrayAdapter<T> asList(T... array)
+    {
+        return ArrayAdapter.adapt(array);
```

Comment:
ArrayAdapter.adapt() already takes varargs.

Reply:
Yes... This addition was purely to provide symmetry with the methods we have on Lists, Sets and Maps.  The method name could either be asList() or adapt().  The method adapt() would be more consistent with the other factory types.  But having a class called Arrays, I thought it might work better to have the same method signature as Arrays.asList() as this could then be used in place of Arrays.asList() which is quite commonly used.

Reply:
We have symmetry in the Iterate classes (Iterate, ListIterate, MapIterate, SetIterate, ArrayIterate, StringIterate).  We did not have full symmetry in the factory classes.

Reply:
@motlin @nikhilnanivadekar  Any thoughts on the naming of asList vs. adapt on Arrays?  I am happy to rename it adapt if you think being more consistent with the other factories is better than being consistent with the java.util.Arrays naming.

Reply:
I don't really like the name Arrays. It is going to be confusing to the users, as it is the same name as Arrays in JDK. Do we really need this symmetry?